src/data/homeSections.json

```diff
@@ -0,0 +1,199 @@
+[
+    {
+        "title": "Latest SDKs",
+        "tag": {
+            "title": "New",
+            "colorClasses": "bg-green-500 text-white"
+        },
+        "img": "/img/star.png",
+        "color": "from-amber-500 to-amber-700",
+        "sdkList": [
+            {
+                "title": "SDK X - Android",
+                "image": "/img/android.png",
+                "alt": "",
```

Comment:
Please add and implement alt text for accessibility reasons. Adding a backlog will also do for now.

Reply:
Ack

Reply:
Done: e923aa0